cowprotocol/cowswap
# feat(analytics): add new dimension for market tracking

# Summary

This PR adds new GA dimension (`dimension5`) which will be current market
"market" is just `${sellSymbol},${buySymbol}::${widgetType}`
Reason for this is that before we were sending this value on some events as a `custom_label` but that is not working in ga4.

![Screenshot 2023-07-25 at 14 08 58](https://github.com/cowprotocol/cowswap/assets/34926005/97019d4c-1262-4869-aefe-d8d87abd64c8)

# To test
1. open any widget 
2. turn on GA Debug extension
3. test any event and check if it has this dimension
4. in case widget type or derived trade is missing, there will be no dimension5 sent with the event, which is expected.
